module-1/lab-data_cleaning/your-code/main.ipynb

```diff
@@ -9,10 +9,21 @@
   },
```

Comment:
1. Step 7: You can do a multiple column rename in just one line using the rename method. It's great for practising your code skills. In the real world,  it's better to use the built-in functions methods/functions in Python and/or third-party library. Only if no built-in method fits your code, you build your "user-defined" method.
2. Step 10: when you use dropping to clean your missing values, it's a good practice to write a chunk of code verifying that before and after applying drop method you are removing only what you want to remove. 
3. Step 11: Have you seen the caveats in your code execution, "Try using .loc[row_indexer,col_indexer] = value instead"? When you try to access to dataframe column for doing some stuff, they are deriving you through loc method instead accesing through indexing (whetever column labels or index)
4. Bonus question: again you get the caveats above, you would need to optimize that chunk of code in this question. There is a way of doing it in the learning unit of data cleaning.